Adding details about notifications and export format

thomasLecler opened the pull request:
## Description

This PR updates the actions/export documentation page :

- Filament export requires notifications to be setup in a panel to be visible in common cases
- All formats are generated when none is provided, then it can be unexpected on big dataset

thomasLecler committed:
Adding details about notifications and export format

Signed-off-by: _Ded@le_ <zeusdieux@hotmail.fr>

danharrin commented on packages/actions/docs/12-export.md:
```diff
@@ -337,7 +339,7 @@ ExportColumn::make('users_avg_age')
 ## Configuring the export formats
 
-By default, the export action will allow the user to choose between both CSV and XLSX formats. You can use the `ExportFormat` enum to customize this, by passing an array of formats to the `formats()` method on the action:
+By default, the export action will generate both CSV and XLSX formats and allow user to choose between it in the notification. You can use the `ExportFormat` enum to customize this, by passing an array of formats to the `formats()` method on the action:
```
```suggestion
By default, the export action will generate both CSV and XLSX formats and allow user to choose between them in the notification. You can use the `ExportFormat` enum to customize this, by passing an array of formats to the `formats()` method on the action:
```

danharrin committed:
Refactor export documentation for clarity

Removed redundant text about publishing migrations for exports and notifications.

Signed-off-by: Dan Harrin <git@danharrin.com>

danharrin committed:
Update packages/actions/docs/12-export.md

Signed-off-by: Dan Harrin <git@danharrin.com>
